docs/logical_data_model.md

```diff
@@ -57,17 +57,72 @@ class RequestErrors {
   * updatedAt : timestamp
 }
 
+class Role {
+  * id : integer
+  * name : string
+}
+
+class Topic {
+  * id : integer
+  * name : string
+}
+
+class RoleTopic {
+  * id : integer <<generated>>
+  * roleId : integer(32) REFERENCES public.Roles.id
+  * topicId: integer(32) REFERENCES public.Topics.id
+  * createdAt : timestamp
+  * updatedAt : timestamp
+}
+
+class Goal {
+  * id : integer
+  * name : string
+  status : string
+  timeframe : string
+  * createdAt : timestamp
+  * updatedAt : timestamp
+}
+
+class TopicGoal {
+  * id : integer
+  * roleId : integer(32) REFERENCES public.Roles.id
+  * topicId: integer(32) REFERENCES public.Topics.id
+  * createdAt : timestamp
+  * updatedAt : timestamp
+}
+
+class Grantee {
+  * id : integer
+  * name : string
+  * createdAt : timestamp
+  * updatedAt : timestamp
+}
+
+class Ttaplan {
+  * id : integer <<generated>>
+  * granteeId : integer(32) REFERENCES public.Grantees.id
+  * grant : string
```

Comment:
Is this the name of the organization receiving the grant? If so, and `granteeId` above becomes `grant.id` we can remove this field.

Reply:
This was refactored and `grant `was removed.